src/commands/kv/bulk/put.rs

```diff
@@ -39,20 +40,35 @@ pub fn put(
         Err(e) => Err(failure::format_err!("{}", e)),
     }?;
 
-    // Validate that bulk upload is within size constraints
-    if pairs.len() > MAX_PAIRS {
-        failure::bail!(
-            "Number of key-value pairs to upload ({}) exceeds max of {}",
-            pairs.len(),
-            MAX_PAIRS
-        );
-    }
+    message::working("Starting KV Bulk Put");
+    let progress_bar = if pairs.len() > MAX_PAIRS {
+        Some(ProgressBar::new(pairs.len() as u64))
+    } else {
+        None
+    };
 
     let client = kv::api_client(user)?;
-    match call_api(&client, target, namespace_id, &pairs) {
-        Ok(_) => message::success("Success"),
-        Err(e) => failure::bail!("{}", kv::format_error(e)),
+    while !pairs.is_empty() {
+        let p: Vec<KeyValuePair> = if pairs.len() > MAX_PAIRS {
+            pairs.drain(0..MAX_PAIRS).collect()
+        } else {
+            pairs.drain(0..).collect()
+        };
+
+        if let Err(e) = call_api(&client, target, namespace_id, &p) {
+            failure::bail!("{}", kv::format_error(e));
+        }
+
+        if let Some(pb) = &progress_bar {
+            pb.inc(p.len() as u64);
+        }
+    }
+
+    if let Some(pb) = &progress_bar {
+        pb.finish_with_message("Done KV Bulk Put");
     }
+
+    message::success("Success");
```

Comment:
probably dont need this line since we have the `uploaded` message w/the progress bar

if you think we should keep it, we should add it to the bulk delete command as well

Reply:
I just left it from before but if you want it gone that works